order-book/src/main/java/com/market/banica/order/book/service/AuroraRequestHandler.java

```diff
@@ -0,0 +1,74 @@
+package com.market.banica.order.book.service;
+
+import com.aurora.Aurora;
+import com.aurora.AuroraServiceGrpc;
+import com.market.banica.order.book.service.grpc.OrderBookService;
+import io.grpc.Status;
+import io.grpc.stub.StreamObserver;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.util.Locale;
+import java.util.regex.Pattern;
+
+@Component
+public class AuroraRequestHandler extends AuroraServiceGrpc.AuroraServiceImplBase {
+    private static final Logger LOGGER = LoggerFactory.getLogger(AuroraRequestHandler.class);
+
+
+    private final OrderBookService orderBookService;
+
+    private static final String SPLIT_SLASH_REGEX = "/+";
+    private static final String SPLIT_EQUALS_REGEX = "=";
+    private static final String NUMBER_CHECK_REGEX = "^\\d+$";
+
+    private static final String SUBSCRIBE = "subscribe";
+
+    private static final Pattern NUMBER_CHECK_PATTERN =
+            Pattern.compile(NUMBER_CHECK_REGEX);
+
+
+    @Autowired
+    public AuroraRequestHandler(OrderBookService orderBookService) {
```

Comment:
is this constructor consistent with other classes' implementation? I believe there was usage of lombok somewhere - this should be approved by the team - are you using constructors or lombok

Reply:
Fixed

Reply:
You can use @RequiredArgsConstructor(onConstructor = @__(@Autowired)), but the whole team follow the convention with constructors. It should be discussed with other team members.